.gitignore

```diff
@@ -19,6 +19,7 @@ _cgo_export.*
 
 _testmain.go
 
+gitbase
```

Comment:
wont this exclude the gitbase folder in cmd?

Reply:
Yeah, also do not include gitignore changes in unrelated commits.

Reply:
Ok, fair enough - I've reverted gitignore.